docs/source/getting_data.rst

```diff
@@ -0,0 +1,242 @@
+
+It is convenient to place all the submissions in a dictionary with their id as the key so that we can retrieve an accepted submission using its id.
+
+We then filter the Decision notes that were accepted and use their forum ID to get the corresponding Submission.
+
+	>>> accepted_submissions = [id_to_submission[note.forum] for note in all_decision_notes if note.content['decision'] == 'Accept']
+
+Retrieving all accepted Submissions for a conference (Double-blind)
+-------------------------------------------------------------------
+This is very similar to the previous example. The only difference is that we need to get the blind notes with the added details parameter to get the Submission.
+
+Retrieve Submissions and Decisions.
+
+	>>> blind_notes = {note.id: note for note in openreview.tools.iterget_notes(client, invitation = 'auai.org/UAI/2019/Conference/-/Blind_Submission', details='original')}
+
+	>>> all_decision_notes = openreview.tools.iterget_notes(client, invitation = 'auai.org/UAI/2019/Conference/-/Paper.*/Decision')
+
+We then filter the Decision notes that were accepted and use their forum ID to get the corresponding Submission.
+
+	>>> accepted_submissions = [blind_notes[decision_note.forum].details['original'] for decision_note in all_decision_notes if 'Accept' in decision_note.content['decision']]
+
+Retrieving all the author names and e-mails from accepted Submissions
+---------------------------------------------------------------------
+First we need to retrieve the Accepted Submissions. Please refer to 'Retrieving all accepted Submissions for a conference'. Once we get the Accepted Submissions we can easily extract the author's information from them.
+
+	>>> author_emails = []
+	>>> author_names = []
+	>>> for submission in accepted_submissions:
+	... 	author_emails += submission['content']['authorids']
+	... 	author_names += submission['content']['authors']
+
+Retrieving comments made on a forum
+----------------------------------------
+
+All comments made on a particular forum/submission can be extracted like this::
+
+	>>>iclr19_forum_comments = client.get_notes(forum="<forum-id>")
+
+Also, the public comments on a particular forum can be extracted like this::
+
+	>>>iclr19_forum_public_comments = client.get_notes(forum="<forum-id>", invitation="ICLR.cc/2019/Conference/-/Paper.*/Public_Comment")
+
+Accessing data in comments
+------------------------------
+
+The data in a comment, or basically Notes objects, can be accessed like this::
+
+	>>>print(iclr19_forum_public_comments[0].content["title"])
+	>>>print(iclr19_forum_public_comments[0].content["comment"])
+
+
+Getting ICLR 2019 data
+--------------------------------
+
+The following example script can be used to retrieve all ICLR 2019 metadata and PDFs::
+
+	import argparse
+	import json
+	import os
+	from collections import defaultdict
+	from tqdm import tqdm
+	import openreview
+
+
+	def download_iclr19(client, outdir='./', get_pdfs=False):
```

Comment:
it seems to be a script here, we could put in the openreview-scripts repo.

Reply:
yeah, I had that thought as well. I though I would propose putting it in `openreview.scripts` so that it could be called from the command line:

```
>>> python -m openreview.scripts.download_iclr19
```